cmd/runtimes/deepspeed/Dockerfile

```
@@ -0,0 +1,42 @@
+FROM nvidia/cuda:12.4.1-devel-ubuntu22.04
```

```suggestion
FROM mpioperator/base:v0.6.0 AS mpi
FROM nvidia/cuda:12.4.1-devel-ubuntu22.04
```

I want to avoid managing sshd config file in this repository.

---

Good suggestion!